app/screens/settings/notification_auto_responder/notification_auto_responder.tsx

```diff
@@ -83,109 +67,90 @@ const NotificationAutoResponder = ({currentUser, componentId}: NotificationAutoR
     const theme = useTheme();
     const serverUrl = useServerUrl();
     const intl = useIntl();
-    const userNotifyProps = useMemo(() => getNotificationProps(currentUser), [currentUser.notifyProps]);
-    const [autoResponderActive, setAutoResponderActive] = useState((currentUser.status === General.OUT_OF_OFFICE && userNotifyProps.auto_responder_active) ? 'true' : 'false');
-    const [autoResponderMessage, setAutoResponderMessage] = useState(userNotifyProps.auto_responder_message || intl.formatMessage(OOO));
+    const notifyProps = useMemo(() => getNotificationProps(currentUser), []);
+
+    const initialAutoResponderActive = useMemo(() => Boolean(currentUser.status === General.OUT_OF_OFFICE && notifyProps.auto_responder_active === 'true'), []);
```

Comment:
Same here, don't we need to fill the dependency array?

Reply:
corrected